packages/backend-plugin-api/api-report.md

```diff
@@ -333,11 +333,19 @@ export interface HttpAuthService {
   }>;
 }
 
+// @public (undocumented)
+export interface HttpRouterHealthCheckConfig {
+  // (undocumented)
+  handler: () => Promise<any>;
```

Comment:
I think it's important to consider a liveness + readiness split. It can help avoid situations where instances are shut down because they timed out at startup, while still being able to be pretty aggressive about timeouts when it comes to whether the service is functional at all.

Reply:
do you have any suggestion for a possible root level readiness implementation?